t/rose-task-run/45-app-fcm-make-compat_mode/suite.rc

```diff
@@ -0,0 +1,16 @@
+#!jinja2
+[cylc]
+    UTC mode = True
+[scheduling]
+    [[dependencies]]
+        graph = """
+            fcm_make:finish => fin
+            fcm_make:fail => fin
+        """
+
+[runtime]
+    [[fcm_make]]
+        script = rose task-run --app-key=fcm_make --new
+    [[fcm_make2]]
```

Comment:
Despite the `fcm_make2` not being in the graph, its config is what the `fcm_make` task is using Cylc Rose to carry out the lookup on, so it needs defined.